src/components/connection-method-selection-dialog.tsx

```diff
@@ -81,13 +83,16 @@ export const ConnectionMethodSelectionDialog: FC<ConnectionMethodSelectionDialog
   methods,
   isShowMobileWarning = true,
   keplrInstallURLOverride,
+  keplrInstallCTAPreset,
   onClose,
   onConnect,
 }) => {
   const [isDialogOpen, setDialogOpen] = React.useState(true);
 
   const isMobile = React.useMemo(isMobileDevice, []);
 
+  const isKeplrUninstalled = !isMobile && !window.keplr;
```

Comment:
Would `isKeplrNotInstalled` or `isKeplrNotExists` be better? `uninstalled` seems strange